ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/electra/block/BlockProcessorElectra.java

```diff
@@ -379,6 +379,23 @@ private void processConsolidationRequest(
     final UInt64 slot = state.getSlot();
     final UInt64 currentEpoch = miscHelpers.computeEpochAtSlot(slot);
 
+    if (isValidSwitchToCompoundingRequest(state, consolidationRequest)) {
+      LOG.debug("process_consolidation_request: valid switching validator to compounding address");
```

Comment:
this would probably be more valuable with context, so that we can see the validator id or abbreviated key?